### PR TITLE
Day2

### PR DESCRIPTION
**Pull request by LuisDavidRodriguez**
### Changes and queries
- Alter table add species Insert 7 animals
- update to unspecified
- ROLLBACK unspecified
ROLLBACK unspecified

--------
- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
- Commit the transaction.
- Verify that change was made and persists after commit.

STEP 2 AND 3

--------
- Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
- After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
STEP 4

--------
- Delete all animals born after Jan 1st, 2022.
- Create a savepoint for the transaction.
- Update all animals' weight to be their weight multiplied by -1.
- Rollback to the savepoint
- Update all animals' weights that are negative to be their weight multiplied by -1.
- Commit transaction
STEP 5 - 1
STEP 5 - 2

--------
- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
queryes

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -7,3 +7,57 @@ VALUES
 ('Pikachu', '07-01-2021', 1, false, 15.04),
 ('Devimon', '12-05-2017', 5, true, 11)
 ;
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+('Charmander', '08-02-2020', 0, false, 11),
```
Comment (zieeco) [inline data.sql]: - Hey, Please, let's make the value for the weight a floating number and not an integer here. `-11.0`

File: data.sql [hunk 2]
```diff
@@ -7,3 +7,57 @@ VALUES
...
+('Plantmon', '15-11-2021', 2, true, 5.7),
+('Squirtle', '02-04-1993', 3, false, 12.13),
```
Comment (zieeco) [inline data.sql:15]: - Hey, Please, let's use the correct value for the weight here. `-12.13`

File: data.sql [hunk 3]
```diff
@@ -7,3 +7,57 @@ VALUES
...
+('Angemon', '12-06-2005', 1, true, 45),
```
Comment (zieeco) [inline data.sql]: - Hey, Please, let's use the correct value and ensure the datatype for the weight is a floating number and not an integer here. `-45.0`

File: data.sql [hunk 4]
```diff
@@ -7,3 +7,57 @@ VALUES
...
+('Boarmon', '07-06-2005', 7, true, 20.4),
+('Blossom', '13-10-1998', 3, true, 17),
```
Comment (zieeco) [inline data.sql]: - Hey, Please, let's make the value for the weight a floating number and not an integer here. `17.0`

File: data.sql [hunk 5]
```diff
@@ -7,3 +7,57 @@ VALUES
...
+('Ditto', '14-05-2022', 4, true, 22);
```
Comment (zieeco) [inline data.sql]: - Hey, Please, let's make the value for the weight a floating number and not an integer here. `22.0`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @LuisDavidRodriguez,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice query and setup. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 5 above)
Comment (LuisDavidRodriguez) [reply data.sql]: Ok dear reviewer that is done

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**